mkosi/mounts.py

```diff
@@ -92,8 +92,6 @@ def finalize_crypto_mounts(config: Config) -> list[PathString]:
     mounts = [
         (root / subdir, Path("/") / subdir)
         for subdir in (
-            Path("usr/share/keyrings"),
```

Comment:
Looking at the docs for `ToolsTreeCertificates=` this is explicitly mentioned there, so that needs to be changed.